recipes/spades/meta.yaml

```diff
@@ -1,33 +1,25 @@
-{% set version = "3.11.1" %}
+{% set version = "3.12.0" %}
 
 package:
   name: spades
   version: {{ version }}
 
 source:
+  # We currently cannot build SPAdes from source, because it requires g++ 5.3.1 or later,
+  # which bioconda does not yet support.  Therefore, we fetch the statically compiled binaries.
   fn: SPAdes-{{ version }}-Darwin.tar.gz  # [osx]
   url: http://cab.spbu.ru/files/release{{ version }}/SPAdes-{{ version }}-Darwin.tar.gz  # [osx]
```

Comment:
You may want to download stuff from github as well. Is there a way to provide several URLs if the primary one is unavailable for one reason or another?

Reply:
@asl There is a way to fetch additional URLs, however only the primary one gets persistently cached by the bioconda setup.   What would the other URLs be?
Typically bioconda recipe URLs point to a tagged release file hosted on github, so that would be best.

Reply:
@asl thanks, changed the URL to github.

The URL in the recipe is normally only accessed once, when the bioconda binary package is built; the package is then uploaded to https://anaconda.org/bioconda/spades/files , and bioconda users fetch it from there.